### ISSUE TITLE
BUG: Nested Vizy fields fail on Safari (desktop and mobile)

### ISSUE THREAD

**Issue by adrienne** [verbb/vizy#17]
**Description**

If I have a Vizy field, and one of the subfields in a Block is also a Vizy field (because i want to use TipTap, and Redactor doesn't work), it fails on iPhone (14.5.1, so not an old version of iOS). Just won't let us type in the subfield at all. Oddly, it works on Android.

**Steps to reproduce**
1. Create a Vizy field called "Blurb" with no blocks (just the TipTap editor)
2. Create a Vizy field called "Content"
3. Add a new block type to Content, and inside that Block type drag the Blurb field
4. Add Content to an entry type
5. Navigate to New Entry on an iPhone
6. Add content to the Content field, then add a block with the Blurb field in it, and try to type in the Blurb

**Additional info**
- Plugin version: 1.0.1
- Craft version: 3.6.13
- Multi-site: Yes but it's not relevant

**Additional context**

**Comment by adrienne** [comment]
@engram-design any ideas on this?

**Comment by engram-design** [comment]
I can replicate this, and certainly strange. Diving into whether this is a tiptap issue, or something with Vizy.

**Comment by adrienne** [comment]
@engram-design - yeah, it's particularly weird since it works on Android standard browsers (which are all Chromium-based), meaning i'm *guessing* it's something specific to iOS-Safari?

**Comment by engram-design** [comment]
Yeah, clearly the case being Safari issue. In fact, I can replicate this behaviour in Safari Desktop (14).

Just trying to isolate if this is an issue with Vizy, TipTap or ProseMirror, and actively working on trying to get a reduced test case together to sort it out.

**Comment by engram-design** [comment]
Getting closer 🤞 https://github.com/ueberdosis/tiptap/issues/1403

**Comment by adrienne** [comment]
@engram-design - awesome! (Using a single editor with views seems like a better design if you can manage it.)

**Comment by engram-design** [comment]
Yeah, not sure if it's that simple, due to having different block types, different editor config, etc - but looking into it!

**Comment by dGilli** [comment]
I have the same experience but on Desktop Safari where nested vizy fields are not editable. I can type and copy into the HTML view but nothing is saved this way.

Seems related.

**Comment by engram-design** [comment]
Yep, it's Safari desktop or mobile.

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.9](https://github.com/verbb/vizy/releases/tag/1.0.9)
